students/AndrewMiotke/lesson01/activity/test_unit.py

```diff
@@ -0,0 +1,83 @@
+from unittest import TestCase
```

Comment:
Great use of MagicMock in these unit tests, nice work!

Reply:
Thanks!